### PR TITLE
API: Fix a typo

### PR DESCRIPTION
**Pull request by enejb**
Introduced in https://github.com/Automattic/jetpack/pull/8654

### CODE REVIEW

File: json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php [hunk 1]
```diff
@@ -89,7 +89,7 @@
 			'$plugin' => '(string) The plugin ID',
 		),
 		'query_parameters'     => array(
-			'invite_accepted' => '(bool=false) If the user is being created in the invite context',
+			'autoupdate' => '(bool=false) If the user is being created in the invite context',
```
Comment (jeherve) [inline json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php]: Shouldn't the string be updated as well?

Comment (enejb) [reply json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php]: Good catch!

**Comment by enejb** [comment]
Updated the description.

**Comment by zinigor** [comment]
Ported into `branch-5.8` in ad89659e0.

**Pull request merged**